releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -313,6 +313,10 @@ private static ReleaseNotesMessage createReleaseNotesMessage(int number, String
         titleField.setAccessible(true);
         titleField.set(issue, title);
 
+        final Field escapedTitleField = issue.getClass().getDeclaredField(" \"`@`\"");
```

Comment:
you need escape function to make code like:
```
escape(issue.getClass().getDeclaredField("title"))

```

Reply:
Changes Done

Reply:
Share prove of testing.

Reply:
I found something strange, the releasenotes-builder runs well with all other release numbers but when I run it for `10.1`,  it just runs and doesn't execute. I waited for a whole day for it to run but it just runs and shows no output ( for eg: `execution succeeded` doesn't come)

Reply:
please focus on writing junit test to test your code, you can prove all execution correctness by junit only.
and later to execution on real release notes.